CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog
```

Comment:
Let's fill this change log

Reply:
Agreed, but I'd prefer to do that in a separate PR